addon/templates/components/bs4/bs-form/element/errors.hbs

```diff
@@ -0,0 +1,3 @@
+{{#if show}}
+  <div class={{feedbackClass}}>{{messages.firstObject}}</div>
+{{/if}}
```

Comment:
It seems the error-templates for `bs3` and `bs4` are still the same, so it could still be a single template in ? common`, right?